assets/xml/objects/object_ani.xml

```diff
@@ -1,61 +1,63 @@
+        <Limb Name="gAniRightForArmLimb" Type="Standard" Offset="0x2840" />
+        <Limb Name="gAniRightHandLimb" Type="Standard" Offset="0x284C" />
+        <Limb Name="gAniHeadLimb" Type="Standard" Offset="0x2858" />
+        <Skeleton Name="gAniSkeleton" Type="Flex" LimbType="Standard" Offset="0x28A0" />
+        <!--- most of these did not load n z64Utils, naming by matchng aganist limbs -->
+        <DList Name="gAniRightHandDL" Offset="0x4860" />
+        <DList Name="gAniRightForArmDL" Offset="0x4A50" />
+        <DList Name="gAniRightUpperArmDL" Offset="0x4B38" />
+        <DList Name="gAniLeftHandDL" Offset="0x4C30" />
+        <DList Name="gAniLeftForArmDL" Offset="0x4E20" />
+        <DList Name="gAniLeftUpperArmDL" Offset="0x4F08" />
+        <DList Name="gAniTorsoDL" Offset="0x5000" />
+        <DList Name="gAniRightFootDL" Offset="0x5270" />
+        <DList Name="gAniRightCalfDL" Offset="0x5420" />
+        <DList Name="gAniRightThighDL" Offset="0x5668" />
+        <DList Name="gAniLeftFootDL" Offset="0x5778" />
+        <DList Name="gAniLeftCalfDL" Offset="0x5928" />
+        <DList Name="gAniLeftThighDL" Offset="0x5C50" />
+        <DList Name="gAniPelvisDL" Offset="0x5D60" />
+        <Texture Name="gAniGeneralTlut" OutName="ani_general_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5E58" />
+        <Texture Name="gAniUnkGradientTex" OutName="ani_unk_gradient1" Format="ci8" Width="8" Height="8" Offset="0x6058" />
+        <Texture Name="gAniFingersTex" OutName="ani_fingers" Format="ci8" Width="8" Height="8" Offset="0x6098" />
+        <Texture Name="gAniShirtEdgeTex" OutName="ani_shirt_edge" Format="i8" Width="8" Height="8" Offset="0x60D8" />
+        <Texture Name="gAniSandleBuckleTex" OutName="ani_sandle_buckle" Format="ci8" Width="16" Height="16" Offset="0x6118" />
+        <Texture Name="gAniPantsTex" OutName="ani_pants" Format="ci8" Width="16" Height="16" Offset="0x6218" />
+        <Texture Name="gAniEyeTlut" OutName="ani_eye_tlut" Format="rgba16" Width="16" Height="12" Offset="0x6318" />
+        <Texture Name="gAniOpenEyeTex" OutName="ani_open_eye" Format="ci8" Width="32" Height="32" Offset="0x6498" TlutOffset="0x6318" />
+        <Texture Name="gAniMouthTex" OutName="ani_mouth" Format="ci8" Width="16" Height="16" Offset="0x6898" />
+        <Texture Name="gAniUnkGradient2Tex" OutName="ani_unk_gradient2" Format="ci8" Width="8" Height="8" Offset="0x6998" />
+        <Texture Name="gAniEarTex" OutName="ani_ear" Format="ci8" Width="8" Height="8" Offset="0x69D8" />
+        <Texture Name="gAniHairTex" OutName="ani_hair" Format="ci8" Width="16" Height="16" Offset="0x6A18" />
+        <Texture Name="gAniClosingEyeTex" OutName="ani_closing_eye" Format="ci8" Width="32" Height="32" Offset="0x6B18" TlutOffset="0x6318"/>
+        <Texture Name="gAniClosedEyeTex" OutName="ani_closed_eye" Format="ci8" Width="32" Height="32" Offset="0x6F18" TlutOffset="0x6318"/>
+        <DList Name="gAnHeadDL" Offset="0x7B58" />
```

Comment:
```suggestion
        <DList Name="gAniHeadDL" Offset="0x7B58" />
```

Reply:
damn it, pretty sure this keyboard I key is dying